### PR TITLE
update docs with note on skip js flag

### PR DESCRIPTION
**Pull request by ConnorStroomberg**
#### Checklist
- [x] User documentation updated
- [x] Clean commits (squash)

### CODE REVIEW

File: docs/developer_documentation/intellij.md [hunk 1]
```diff
@@ -219,3 +219,15 @@ Now, whenever you make changes to one or more JavaScript files, the correspondin
 the npm watch task. Once it's built, tab out of IntelliJ to trigger a refresh of the exploded war.
 As soon as IntelliJ loses focus, you'll see a task progress bar in the bottom right corner of IntelliJ.
 Wait for that task to finish and then refresh the browser. The changes will be loaded.
+
+## Speed up building the project by skipping the javascript build
+
+Building the project can be sped up by not building the javascript. To skip building the javascript use add:
+
+`-Dskip.js=true`
```
Comment (dennishendriksen) [inline docs/developer_documentation/intellij.md]: What is the difference with using the profile ```skip-js-compile```?

Comment (ConnorStroomberg) [reply docs/developer_documentation/intellij.md]: none , `skip.js=true` set the flag , that triggers the profile

File: docs/developer_documentation/intellij.md [hunk 2]
```diff
@@ -219,3 +219,15 @@ Now, whenever you make changes to one or more JavaScript files, the correspondin
 the npm watch task. Once it's built, tab out of IntelliJ to trigger a refresh of the exploded war.
 As soon as IntelliJ loses focus, you'll see a task progress bar in the bottom right corner of IntelliJ.
 Wait for that task to finish and then refresh the browser. The changes will be loaded.
+
+## Speed up building the project by skipping the javascript build
+
+Building the project can be sped up by not building the javascript. To skip building the javascript use add:
```
Comment (tommydeboer) [inline docs/developer_documentation/intellij.md]:
```suggestion
Building the project can be sped up by not building the javascript. To skip building the javascript, use:
```

File: docs/developer_documentation/intellij.md [hunk 3]
```diff
@@ -219,3 +219,15 @@ Now, whenever you make changes to one or more JavaScript files, the correspondin
...
+
+`-Dskip.js=true`
+
+to the maven build command.
+
+* Note: 
+    - If javascript was not already build, javascript will be missing and the application will not function properly.
+    - If changes where made to de javascript these changes will not be build into the application ( as building will be skipped).
```
Comment (tommydeboer) [inline docs/developer_documentation/intellij.md]:
```suggestion
    - If changes were made to the javascript these changes will not be built into the application (as building will be skipped). 
```

File: docs/developer_documentation/intellij.md [hunk 4]
```diff
@@ -219,3 +219,15 @@ Now, whenever you make changes to one or more JavaScript files, the correspondin
...
+## Speed up building the project by skipping the javascript build
+
+Building the project can be sped up by not building the javascript. To skip building the javascript use add:
+
+`-Dskip.js=true`
+
+to the maven build command.
+
+* Note: 
+    - If javascript was not already build, javascript will be missing and the application will not function properly.
```
Comment (tommydeboer) [inline docs/developer_documentation/intellij.md]:
```suggestion
    - If javascript was not already built, javascript will be missing and the application will not function properly.
```

**Pull request merged**